tb_plugin/README.md

```diff
@@ -45,3 +45,98 @@
 
   If the files under `--logdir` are too big or too many.
   Refresh the browser to check latest loaded result.
+
+### Quick Usage Instructions
```

Comment:
need to update some terms and images after renaming.

Reply:
Agree.